ksqldb-engine/src/main/java/io/confluent/ksql/function/udaf/offset/LatestByOffset.java

```diff
@@ -40,43 +45,75 @@
 public final class LatestByOffset {
 
   static final String DESCRIPTION =
-      "This function returns the most recent value for the column, computed by offset.";
+      "This function returns the most recent n values for the column, computed by offset.";
 
   private LatestByOffset() {
   }
 
   static AtomicLong sequence = new AtomicLong();
+  
+  static final Comparator<Struct> structComparator  = new LatestStructComparator();
 
   @UdafFactory(description = "return the latest value of an integer column",
       aggregateSchema = "STRUCT<SEQ BIGINT, VAL INT>")
   public static Udaf<Integer, Struct, Integer> latestInteger() {
     return latest(STRUCT_INTEGER);
   }
+  
+  @UdafFactory(description = "return the latest value of an integer column",
+      aggregateSchema = "ARRAY<STRUCT<SEQ BIGINT, VAL INT>>")
+  public static Udaf<Integer, List<Struct>, List<Integer>> latestInteger(final int topNSize) {
+    return latestN(STRUCT_INTEGER, topNSize);
+  }
```

Comment:
Same changes to the new functions in this file, as were done to the ones in `EarliestByOffset`, i.e. plural method names and param names changes to `latestN`.